packages/vow/test/test-disconnect.js

```diff
@@ -52,18 +52,18 @@ test('retry on disconnection', async t => {
     [2, 'disco', 'disco', 'sad'],
   ];
 
-  for await (const watchVow of [false, true]) {
-    t.log('testing watchVow', watchVow);
+  for await (const doWatch of [false, true]) {
+    t.log('testing watch', doWatch);
```

Comment:
Suggestion: replace these boolean values with descriptive strings.
```suggestion
  for await (const pattern of ['when', 'watch']) {
    t.log('testing', pattern);
```